BUILDING.md

```diff
@@ -89,7 +92,7 @@ Depending on host platform, the selection of toolchains may vary.
 
 #### Windows
 
-* Visual Studio 2017 or the Build Tools thereof
+* Visual Studio 2017 with the Windows 10 SDK on a 64 bit host.
```

Comment:
This is under "Supported Toolchains". There is no need to mention Windows 10 SDK or bitness here. The specific requirements are listed in a separate section. Why change this line at all?

Reply:
IMHO SDK and host bitness are parts of the toolchain 🤷‍♂️